### PR TITLE
added CartItemRepositoryTest

### PR DESCRIPTION
**Pull request by yanna-stepanova**

### CODE REVIEW

**Comment by yanna-stepanova** [comment]
Screenshot 2024-06-20 195237
Screenshot 2024-06-20 200126

File: src/test/java/com/yanna/stepanova/service/ShoppingCartServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    @Test
+    @DisplayName("""
+            Get exception for non-existing user id""")
+    public void getShopCart_NonExistingId_ReturnException() {
+        // given
+        Long userId = 2L;
+        String expected = "Can't find shopping cart by id:" + userId;
+        Mockito.when(shoppingCartRepo.findById(userId)).thenReturn(Optional.empty());
+        // when
+        try {
```
Comment (okuzan) [inline src/test/java/com/yanna/stepanova/service/ShoppingCartServiceTest.java:55]:
```suggestion

        // when
        try {
```
better leave an empty line between given/when/then blocks

File: src/test/java/com/yanna/stepanova/service/ShoppingCartServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    @Mock
+    private ShoppingCartRepository shoppingCartRepo;
+    @Mock
+    private ShoppingCartMapper shoppingCartMapper;
+    @InjectMocks
+    private ShoppingCartServiceImpl shoppingCartService;
+
+    @Test
+    @DisplayName("""
+            Get shopping cart by user id""")
```
Comment (okuzan) [inline src/test/java/com/yanna/stepanova/service/ShoppingCartServiceTest.java]:
```suggestion
    @DisplayName("Get shopping cart by user id")
```
no need for text blocks for one line names

File: src/test/java/com/yanna/stepanova/service/ShoppingCartServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        } catch (EntityNotFoundException exception) {
+            // then
+            Assertions.assertEquals(expected, exception.getMessage());
+        }
+    }
+
+    @Test
+    @DisplayName("""
+            Get correct ShoppingCartDto by user id""")
+    void getShopCartDto_WithValidId_ReturnShoppingCartDto() {
```
Comment (okuzan) [inline src/test/java/com/yanna/stepanova/service/ShoppingCartServiceTest.java]: most of your methods are public, this one is not - unify this

**Review by okuzan — APPROVED** [review]
Well done, some code style suggestions

**Pull request merged**